src/Models/Model.js

```diff
@@ -0,0 +1,15 @@
+import {NewsChannel} from '../js/component/NewsChannel';
+export class Model{
+    constructor(sources) {
+        this.sources = sources
+        this.channels = {}
+    }
+    getChannels() {        
+        this.channels = this.sources.reduce((acc, { source, inst }) => {
+            acc[inst] = new NewsChannel(source, this.something);
+            return acc
+          }, {})
+    }
```

Comment:
So as for Model it should be a data storage for your application. You need to fetch data inside of Controller, store it in Model and render it using View

Reply:
The root cause why you your items are not rendered in your renderItem method. It becomes a kind of a mess and I would suggest to rethink what you will have in pieces of your application. 
Model will store your channels and articles for that. So that should be data storage of your application.

With Controller you can make requests, attach/detouch events, store all logic here. 

View - it is all about DOM manipulations. All things regarding renders should be here.

You do not need NewsChannel class anymore.

Please read that carefully and try to apply that changes